src/enum/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
```

Comment:
You use it with filters, so the name can also be associated with words close to the filter

Reply:
Agree